x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -67,192 +82,201 @@ export function AddDataView() {
+        key: AddDataTabs.Kubernetes,
+        title: i18n.translate('xpack.profiling.tabs.kubernetesTitle', {
+          defaultMessage: 'Kubernetes',
+        }),
+        steps: [
+          {
+            title: i18n.translate('xpack.profiling.tabs.kubernetesRepositoryStep', {
+              defaultMessage: 'Configure the Universal Profiling Agent Helm repository:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {i18n.translate('xpack.profiling.tabs.helmRepoAddElasticCodeBlockLabel', {
+                  defaultMessage: 'helm repo add elastic https://helm.elastic.co',
+                })}
+              </EuiCodeBlock>
+            ),
+          },
+          {
+            title: i18n.translate('xpack.profiling.tabs.kubernetesInstallStep', {
+              defaultMessage: 'Install host-agent via Helm:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {`helm install --create-namespace -n=universal-profiling universal-profiling-agent \\
 --set "projectID=1,secretToken=${secretToken}" \\
 --set "collectionAgentHostPort=${collectionAgentHost}" \\
 --set "version=${stackVersion}" \\
 --version=${stackVersion} \\
 elastic/pf-host-agent`}
-            </EuiCodeBlock>
-          ),
-        },
-        {
-          title: i18n.translate('xpack.profiling.tabs.kubernetesValidationStep', {
-            defaultMessage: 'Validate the host-agent pods are running:',
-          }),
-          content: (
-            <EuiCodeBlock paddingSize="s" isCopyable>
-              kubectl -n universal-profiling get pods
-            </EuiCodeBlock>
-          ),
-        },
-        {
-          title: i18n.translate('xpack.profiling.tabs.postValidationStep', {
-            defaultMessage:
-              'Use the Helm install output to get host-agent logs and spot potential errors',
-          }),
-          content: <></>,
-        },
-      ],
-    },
-    {
-      key: AddDataTabs.Docker,
-      title: i18n.translate('xpack.profiling.tabs.dockerTitle', {
-        defaultMessage: 'Docker',
-      }),
-      steps: [
-        {
-          title: i18n.translate('xpack.profiling.tabs.dockerRunContainerStep', {
-            defaultMessage: 'Run the Universal Profiling container:',
-          }),
-          content: (
-            <EuiCodeBlock paddingSize="s" isCopyable>
-              {`docker run --name pf-host-agent --privileged --pid=host -v /etc/machine-id:/etc/machine-id:ro \\
+              </EuiCodeBlock>
+            ),
+          },
+          {
+            title: i18n.translate('xpack.profiling.tabs.kubernetesValidationStep', {
+              defaultMessage: 'Validate the host-agent pods are running:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {i18n.translate(
+                  'xpack.profiling.tabs.kubectl-nUniversal-profilingGetCodeBlockLabel',
```

Comment:
@inge4pres please can you Remove the `-`, and create a better name for this key and all keys that were automatically generated? This is the reason why the CI is failing.

Reply:
I changed them but there is still an error that i can't figure out how to fix 🤔